app/views/posts/_user_recommendations.erb

```diff
@@ -0,0 +1,9 @@
+<% cache [current_locale, current_user.recommended_posts] do %>
```

Comment:
Does `current_user.recommended_posts` have a good equality check for the cache?

Reply:
It should do! It is just an active records collection so it should work like any other